sympy/series/formal.py

```diff
@@ -972,6 +980,11 @@ def infinite(self):
 
         return self.ind + inf_sum
 
+    @property
+    def free_symbols(self):
+        """Returns the free symbols of the formal power series"""
+        return self.function.free_symbols
```

Comment:
I checked we  already have this property implemented in ``SeriesBase``. Checkout the implementation to see how to implement ``free_symbols`` correctly. Remove this.